### ISSUE TITLE
NameError: uninitialized constant DataShift::LoaderBase::Forwardable

### ISSUE THREAD

**Issue by flyfy1** [autotelik/datashift#51]
Version: current latest on RubyGems, `0.40.0`

Script: 

```ruby
require 'thor'
require 'datashift'
```

Error Message:

```
Problem initializing gem #<NameError: uninitialized constant DataShift::LoaderBase::Forwardable>
Problem loading datashift file /Users/songyy/.rvm/gems/ruby-2.3.1/gems/datashift-0.40.0/lib/exporters/csv_exporter.rb - #<NameError: uninitialized constant DataShift::ExporterBase
Did you mean?  DataShift::Exporters>
Problem loading datashift file /Users/songyy/.rvm/gems/ruby-2.3.1/gems/datashift-0.40.0/lib/exporters/excel_exporter.rb - #<NameError: uninitialized constant DataShift::ExporterBase
Did you mean?  DataShift::Exporters>
```

**Comment by autotelik** [comment]
Strange I don't see this, maybe a path load issue, I'm on Ubuntu

I will add a specific require to LoaderBase

for time being adding require 'forwardable' at top of your script should resolve this

**Comment by flyfy1** [comment]
Thx @autotelik . The issue seems to be resolved after version 0.40.1

**Comment by autotelik** [comment]
great, thanks for the feedback
